django/contrib/gis/gdal/srs.py

```diff
@@ -116,7 +116,11 @@ def __getitem__(self, target):
         doesn't exist.  Can also take a tuple as a parameter, (target, child),
         where child is the index of the attribute in the WKT.  For example:
 
-        >>> wkt = 'GEOGCS["WGS 84", DATUM["WGS_1984, ... AUTHORITY["EPSG","4326"]]'
+        >>> wkt = (
+        ...     'GEOGCS["WGS 84",'
+        ...     '  DATUM["WGS_1984, ... AUTHORITY["EPSG","4326"]'
+        ...     ']'
+        ... )
```

Comment:
I'm not thrilled with this editorial change, but it's equivalent to the original. (Given the syntax error in the original, I suspect it's psuedo-code anyway.)